### PR TITLE
Create HelloWorld.cpp

### PR DESCRIPTION
**Pull request by torablien**
We are missing a programmer's greetings!

### CODE REVIEW

**Comment by torablien** [comment]
@theyounggriff What do you think?

File: personal-files/HelloWorld.cpp [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+
+int main(void)
```
Comment (theyounggriff) [inline personal-files/HelloWorld.cpp]: Nice job on this line!

**Comment by theyounggriff** [comment]
Overall looks good!

File: personal-files/Intro.md [hunk 2]
```diff
@@ -2,3 +2,5 @@
 Hi there,
 
 My name is Neil Barot, and I am a freshman at Tulane University!
+
+My favorite resteraunt in New Orleans is Reginelli's Pizzeria. You should definitely try the Uptowner!
```
Comment (torablien) [inline personal-files/Intro.md]: Added my favorite restaurant in New Orleans!

**Pull request merged**